packages/i18n-lint/src/verifiers/__tests__/jsonVerifier.spec.js

```diff
@@ -0,0 +1,19 @@
+import { validateJson } from '../jsonVerifier';
+
+jest.unmock('../jsonVerifier');
+
+describe('jsonVerifier', () => {
+  it('should detect that two same level keys (first range) are not sorted', () => {
+    const invalidJson = {
+      b: 'bad position',
+      a: {
+        z: 'bad',
+        c: 'position',
+      },
+    };
+    expect(validateJson(invalidJson)).toEqual([
+      { error: true, message: 'current key b is bigger than its next key a' },
```

Comment:
If you send this to the report builder, I think the result will be a little bit strange  🤔

Reply:
I'm not sure to get the idea, but it's the actual message from the reportbuilder that is tested right there (integration), see https://github.com/M6Web/i18n-tools/pull/4/files#diff-4bbc7bb7c6c65b70b0bd14728b5abe55R8